mazihang/client/src/client.cpp

```diff
@@ -0,0 +1,38 @@
+#include "ros/ros.h"
+#include "client/time.h"
+
+#include "time.h"
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    struct tm *local;
+    time_t t;
+    t=time(NULL);
+    local=localtime(&t);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Consider using localtime_r(...) instead of localtime(...) for improved thread safety.  [runtime/threadsafe_fn] [2]